sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/transforms/SampleTest.java

```diff
@@ -258,4 +261,15 @@ public void testPickAnyWhenEmpty() {
   public void testSampleGetName() {
     assertEquals("Sample.SampleAny", Sample.<String>any(1).getName());
   }
+
+  @Test
+  public void testDisplayData() {
+    PTransform<?, ?> sampleAny = Sample.any(1234);
+    DisplayData sampleAnyDisplayData = DisplayData.from(sampleAny);
+    assertThat(sampleAnyDisplayData, hasDisplayItem("sampleSize", 1234));
```

Comment:
`Sample` doesn't have an override for estimation error. `ApproximateUnique` does, which I've added display data + tests for.